src/http/span.rs

```diff
@@ -208,6 +178,20 @@ pub(crate) fn parse_response_from_bytes(
     Ok(response)
 }
 
+/// Converts a `httparse::Header` to a `Header`.
+fn from_header(src: &Bytes, header: &httparse::Header) -> Header {
+    let name_range = get_span_range(src, header.name.as_bytes());
+    let value_range = get_span_range(src, header.value);
+    // Add 2 to the end of the range to account for the terminating \r\n.
+    let header_range = name_range.start..value_range.end + 2;
```

Comment:
` header_range` must also account for any optional trailing whitespaces that may exist between the value and CRLF.